javascript.ru/2.12.4/2.12.4.js

```diff
@@ -0,0 +1,22 @@
+﻿var user = prompt('Who are you?');
+
+if (user == 'Admin') {
```

Comment:
What about case insensitive check for all `if` except `if (password == 'Loki')`?

Reply:
Done a9e11ec.